test/unit/resources/http_test.rb

```diff
@@ -243,4 +243,23 @@
       _(headers.D).must_equal(4)
     end
   end
+
+  describe "Windows" do
+
+    # windows
+    it "simple HTTP request with no options" do
+      resource = MockLoader.new(:windows).load_resource("http", "https://www.example.com")
```

Comment:
The issue with the test is it seems to be defaulting to local while the code (currently) only works for remote. 
How can I change the test to use the correct backend?